# Type: docs
# Subtype: D2
# English Summary: Added a description of package loading behavior for multiple parent class loaders
# Chinese Summary: 为多个父类加载器添加了一个关于包加载行为的描述
# Entry Number: 6

# === Git Diff Content ===

diff --git a/byte-buddy-dep/src/main/java/net/bytebuddy/dynamic/loading/MultipleParentClassLoader.java b/byte-buddy-dep/src/main/java/net/bytebuddy/dynamic/loading/MultipleParentClassLoader.java
index cbfef42925..02ae7e557d 100644
--- a/byte-buddy-dep/src/main/java/net/bytebuddy/dynamic/loading/MultipleParentClassLoader.java
+++ b/byte-buddy-dep/src/main/java/net/bytebuddy/dynamic/loading/MultipleParentClassLoader.java
@@ -26,6 +26,11 @@ import static net.bytebuddy.utility.ByteBuddyCommons.filterUnique;
  * For the {@link java.lang.ClassLoader#getResources(java.lang.String)} method, this means that this class loader
  * might return the same url multiple times by representing the same class loader multiple times.
  * </p>
+ * <p>
+ * <b>Important</b>: This class loader does not support the location of packages from its multiple parents. This breaks
+ * package equality when loading classes by either loading them directly via this class loader (e.g. by subclassing) or
+ * by loading classes with child class loaders of this class loader.
+ * </p>
  */
 public class MultipleParentClassLoader extends ClassLoader {
 
